public/app/core/components/Upgrade/UpgradeBox.tsx

```diff
@@ -88,11 +79,151 @@ const getUpgradeBoxStyles = (theme: GrafanaTheme2, size: ComponentSize) => {
+          {listItems.map((item, index) => (
+            <li key={index}>
+              <Icon name={'check'} size={'xl'} className={styles.icon} /> {item}
+            </li>
+          ))}
+        </ul>
+        {featureUrl && (
+          <LinkButton fill={'text'} href={featureUrl} className={styles.link}>
+            Learn more
+          </LinkButton>
+        )}
+      </div>
+      <div className={styles.media}>
+        <img src={getImgUrl(image)} alt={'Feature screenshot'} />
+        {caption && <p className={styles.caption}>{caption}</p>}
+      </div>
+    </div>
+  );
+};
+
+const getUpgradeContentStyles = (theme: GrafanaTheme2) => {
+  return {
+    container: css`
+      display: flex;
+      justify-content: space-between;
+    `,
+    content: css`
+      width: 45%;
+      margin-right: ${theme.spacing(4)};
+    `,
+    media: css`
+      width: 55%;
+
+      img {
+        width: 100%;
+      }
+    `,
+    description: css`
+      color: ${theme.colors.text.primary};
+      font-weight: ${theme.typography.fontWeightLight};
+    `,
+    list: css`
+      list-style: none;
+      margin: ${theme.spacing(4, 0, 2, 0)};
+
+      li {
+        display: flex;
+        align-items: flex-start;
+        color: ${theme.colors.text.primary};
+        padding: ${theme.spacing(1, 0)};
+      }
+    `,
+    icon: css`
+      color: ${theme.colors.success.main};
+      margin-right: ${theme.spacing(1)};
+    `,
+    link: css`
+      margin-left: ${theme.spacing(2)};
+    `,
+    title: css`
+      color: ${theme.colors.text.maxContrast};
+    `,
+    caption: css`
+      font-weight: ${theme.typography.fontWeightLight};
+      margin: ${theme.spacing(1, 0, 0)};
+    `,
+  };
+};
+
+export const UpgradeContentVertical = ({
+  featureName,
+  description,
+  featureUrl,
+  image,
+}: Omit<UpgradeContentProps, 'listItems' | 'caption'>) => {
+  const styles = useStyles2(getContentVerticalStyles);
+  return (
+    <div>
+      <h3 className={styles.title}>Get started with {featureName}</h3>
+      {description && <h6 className={styles.description}>{description}</h6>}
+      <LinkButton fill={'text'} href={featureUrl}>
+        Learn more
+      </LinkButton>
+      <div className={styles.media}>
+        <img src={getImgUrl(image)} alt={'Feature screenshot'} />
+      </div>
+    </div>
+  );
+};
+
+const getContentVerticalStyles = (theme: GrafanaTheme2) => {
+  return {
+    title: css`
+      color: ${theme.colors.text.maxContrast};
+    `,
+    description: css`
+      color: ${theme.colors.text.primary};
+      font-weight: ${theme.typography.fontWeightLight};
+    `,
+    media: css`
+      width: 100%;
+      margin-top: ${theme.spacing(2)};
+
+      img {
+        width: 100%;
+      }
+    `,
+  };
+};
+
+// Get Gdrive URL by item id
+const getImgUrl = (urlOrId: string) => {
+  if (urlOrId.startsWith('http')) {
+    return urlOrId;
+  }
+
+  return 'https://drive.google.com/uc?export=download&id=' + urlOrId;
```

Comment:
This doesn't work if you don't have access to the image in Google Drive and I don't think it's a good practice. We should either save them in `public/img/...` (maybe with smaller resolution) or find another solution with the Hosted Grafana team.